src/applications/appeals/10182/constants.js

```diff
@@ -69,7 +69,7 @@ export const SUBMITTED_DISAGREEMENTS = {
   evaluation: 'disability evaluation',
 };
 
-export const LAST_ISSUE = 'lastNodItem'; // focus management across pages
+export const LAST_ISSUE = 'lastItem'; // focus management across pages
```

Comment:
Is it okay to rename this in both 10182 and 996 constants? 
I was hesitant to move to shared constants to avoid rabbit hole again.

Reply:
Since this is a key for session storage and only used while switching between the contestable issues widget and the add issue page to manage focus, I think it'll be okay to eventually use the same name (`lastItem`) and move it to the shared constants file.

Reply:
Roger that. Will do so when refactoring constants in future ticket.